docs/user-guide/volumes.md

```diff
@@ -336,7 +336,7 @@ See the [GlusterFS example](../../examples/glusterfs/) for more details.
 An `rbd` volume allows a [Rados Block
 Device](http://ceph.com/docs/master/rbd/rbd/) volume to be mounted into your
 pod.  Unlike `emptyDir`, which is erased when a Pod is removed, the contents of
-an `rbd` volume are preserved and the volume is merely unmounted.  This
+a `rbd` volume are preserved and the volume is merely unmounted.  This
```

Comment:
I stand corrected. http://english.stackexchange.com/questions/50851/the-contents-are-or-the-contents-is